interns-react-weekly-assignments/src/components/Home/Ritika/Week4/components/DeleteModal/DeleteModal.styled.ts

```diff
@@ -0,0 +1,81 @@
+import styled from 'styled-components';
+
+export const DeleteModalWrapper = styled.div`
+
+margin: auto;
+width: 100%;
+position: fixed;
+backdrop-filter: blur(5px);
+height: 100%;
+align-items: center;
+display: grid;
+
+
+.add-task{
```

Comment:
I can see you switching between different formats for classnNames, please choose one